src/widgets/events/ui/events.test.tsx

```diff
@@ -1,6 +1,6 @@
 import { screen } from '@testing-library/react';
 import { describe, expect, it } from 'vitest';
-import { Events } from './ui/events';
+import { Events } from './events.tsx';
```

Comment:
Was it vs code that messed up with updating imports?

Reply:
Sorry, my JB IDE really likes to add extensions to all imports. I will remove this rule so that there will be no such errors in the future.